metrics/metrics.go

```diff
@@ -48,6 +48,19 @@ var (
 		},
 	)
 
+	DuplicatePods = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "pods_duplicate",
+			Help:           "Number of duplicate pods on a node",
```

Comment:
You mean duplicates as they are defined by RemoveDuplicates strategy? Or, is the metrics supposed to include duplicates of other strategies (in the future) as well?

Reply:
Ya, I want to include statefulsets in future.